lib/Setup Thermostats.js

```diff
@@ -181,6 +183,7 @@ exports.setupThermostat = function (that, services)
 			.updateOnHubEvents(that.currentAccessory.id, "coolingSetpoint")
 			.on('set', function(newHomekitValue, callback, context)
 				{
+					if(TargetState.value !== 3) return callback(null);

```

Comment:
I think this and the prior change are the only ones that really matter to get the effect you are looking for.